### PR TITLE
Create beautiful page

### PR DESCRIPTION
**Pull request by annagigoian**
Very beautiful.
Reference https://content.codecademy.com/courses/freelance-1/unit-2/dasmotos-arts_redline.jpg

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,91 @@
...
- 
-</html>
+
+<footer>
+  <p>
+    &copy; Copyright Bip-Boop Development, LTD. All Rights Reserved.
+  </p>
+</footer>
+
+</html>
```
Comment (mikhailmikhaylov) [inline index.html:91]: Please setup automatic new line on save

File: resources/index.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    padding: 10px;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-size: 16px;
+    font-weight: bold;
+    margin-top: 20px;
+}
+ button:hover {
+     background-color: indigo;
+ }
```
Comment (mikhailmikhaylov) [inline resources/index.css:85]: Here too

**Comment by annagigoian** [comment]
Take a look again
pleaesedsedes

**Comment by mikhailmikhaylov** [comment]
looks good to me

**Pull request merged**